src/data-generator/declare.ts

```diff
@@ -218,6 +218,7 @@ export async function createBirthDeclaration(
 
 export async function createDeathDeclaration(
   { username, token }: User,
+  deathTime: Date,
```

Comment:
Death declarations now also support configurable distribution of completeness rates